compiler/x/codegen/CompareAnalyser.cpp

```diff
@@ -730,8 +730,7 @@ void TR_X86CompareAnalyser::longOrderedCompareAndBranchAnalyser(TR::Node       *
    cg()->decReferenceCount(firstChild);
    cg()->decReferenceCount(secondChild);
 
-   if (deps)
-      deps->setMayNeedToPopFPRegisters(true);
+   deps->setMayNeedToPopFPRegisters(true);
```

Comment:
I don't doubt this one is ok - it just seemed odd to fix it in only one case in the method, but if that is what you prefer to do I'm ok with it. I was just pointing out there may be more that can be tidied up.